src/test/java/org/gridsuite/shortcircuit/server/repositories/FeederResultRepositoryTest.java

```diff
@@ -293,23 +311,23 @@ private Stream<Arguments> provideGreaterThanOrEqualFilters() {
                 resultMagnitudeEntity,
                 List.of(
                     new ResourceFilter(ResourceFilter.DataType.NUMBER, ResourceFilter.Type.GREATER_THAN_OR_EQUAL, 22.17, "current")),
-                List.of(feederResultEntity1, feederResultEntity3)),
+                feederResultEntityMagnitudeList.stream().filter(feederResultEntity -> Double.compare(feederResultEntity.getCurrent(), 22.17) >= 0).toList()),
             Arguments.of(
                 resultMagnitudeEntity,
                 List.of(
                     new ResourceFilter(ResourceFilter.DataType.NUMBER, ResourceFilter.Type.GREATER_THAN_OR_EQUAL, 18.56, "current")),
-                List.of(feederResultEntity1, feederResultEntity2, feederResultEntity3)),
+                feederResultEntityMagnitudeList.stream().filter(feederResultEntity -> Double.compare(feederResultEntity.getCurrent(), 18.56) >= 0).toList()),
             Arguments.of(
                 resultMagnitudeEntity,
                 List.of(
                     new ResourceFilter(ResourceFilter.DataType.NUMBER, ResourceFilter.Type.GREATER_THAN_OR_EQUAL, 53.95, "current")),
-                List.of()),
+                feederResultEntityMagnitudeList.stream().filter(feederResultEntity -> Double.compare(feederResultEntity.getCurrent(), 53.95) >= 0).toList()),
             Arguments.of(
                 resultMagnitudeEntity,
                 List.of(
                     new ResourceFilter(ResourceFilter.DataType.NUMBER, ResourceFilter.Type.GREATER_THAN_OR_EQUAL, 22.17, "current"),
                     new ResourceFilter(ResourceFilter.DataType.NUMBER, ResourceFilter.Type.GREATER_THAN_OR_EQUAL, 53.94, "current")),
-                List.of(feederResultEntity3))
+                feederResultEntityMagnitudeList.stream().filter(feederResultEntity -> Double.compare(feederResultEntity.getCurrent(), 53.94) >= 0).toList())
```

Comment:
Yes it was necessary, all the tests basically failed with the changes in order because we put the result in a specific order ourselves in the tests instead of finding the right result dynamically. So to avoid redoing the tests all over again when something changes in the sorting I needed to do this changes.